### PR TITLE
Immersive Pride Month

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Requires this PR https://github.com/Citadel-Station-13/Citadel-Station-13/pull/12436

Basically makes the pride month holiday immersive and realistic, by making brands exploit it. On pride month, all vending machines have a rainbow overlay and phrases about how important your sexuality is to them, and of course the prices are all changed to be 20% more expensive.

**It's just like real life!**

## Why It's Good For The Game

image

## Changelog
:cl:
add: Pride month is now realistic and immersive.
/:cl:

### CODE REVIEW

**Comment by kappa-sama** [comment]
@deathride58

**Comment by deathride58** [comment]
Good first PR

**Comment by nik707** [comment]
10 out of 10

File: code/modules/vending/cola.dm [hunk 1]
```diff
@@ -107,4 +125,7 @@
 	premium = list(/obj/item/reagent_containers/food/drinks/beer/light = 1,
 					/obj/item/reagent_containers/food/drinks/soda_cans/grey_bull = 1)
 	product_slogans = "~A hive of Flavour!~"
-	product_ads = "Stinging flavour!;Ideal of the worker drone!;Buzz Buzz!;Got Honey?;The Queen approves of this drink!;Sap back that missing energy!;Bee the consumer unit!"
+	product_ads = "Stinging flavour!;Ideal of the worker drone!;Buzz Buzz!;Got Honey?;The Queen approves of this drink!;Sap back that missing energy!;Bee the consumer unit!;1, 2, Buzz, 4, Fuzz, Buzz!" //Skyrat edit - fizz buzz funny joke
+	//skyrat edit
+	brand = "Buzz Fuzz"
```
Comment (Trilbyspaceclone) [inline code/modules/vending/cola.dm:130]:
```suggestion
	brand = "Shambler's Drinks"
```
Lore wise its the same people

Comment (BobJoga) [reply code/modules/vending/cola.dm:130]: it was a coder joke

Comment (BobJoga) [reply code/modules/vending/cola.dm:130]: oh wait hold up

**Comment by TheSpaghetti** [comment]
trash PR, Pride Month was a mistake

**Comment by BobJoga** [comment]
> trash PR, Pride Month was a mistake

image

**Comment by jjpark-kb** [comment]
I feel that this PR was not made in good favor of the pride month PR. I find it somewhat hilarious, but it is somewhat bad.

**Pull request closed without merging**